docs/docs/cmd/pa/app/app-list.md

```diff
@@ -0,0 +1,32 @@
+# pa app list
+
+Lists apps
```

Comment:
I've updated the description to `Lists all Power Apps apps` to make it more explicit that it's about Power Apps apps :)